### ISSUE TITLE
Postgres extention on Rstudio

### ISSUE THREAD

**Issue by zcatav** [duckdb/duckdb-r#624]
### What happens?

Duckdb 1.1.3 is installed on R 4.4.2, and when I try to install the Postgres extension, it gives an error.

```R
library(duckdb)
con <- dbConnect(duckdb())

dbExecute(con, "INSTALL postgres;")
```
```
Error in `duckdb_result()`:
! rapi_execute: Failed to run query
Error: IO Error: Failed to install 'postgres_scanner'
The file was built for DuckDB version 'v1.1.3', but we can only load extensions built for DuckDB version '19864453f7'.
Run `rlang::last_trace()` to see where the error occurred.`
```

There is no problem when performing operations on the command line.

### To Reproduce

```R
dbExecute(con, "INSTALL postgres;")
```
```console
Error in `duckdb_result()`:
! rapi_execute: Failed to run query
Error: IO Error: Failed to install 'postgres_scanner'
The file was built for DuckDB version 'v1.1.3', but we can only load extensions built for DuckDB version '19864453f7'.
Run `rlang::last_trace()` to see where the error occurred.
```

```R
rlang::last_trace()
```
```console
<error/rlang_error>
Error in `duckdb_result()`:
! rapi_execute: Failed to run query
Error: IO Error: Failed to install 'postgres_scanner'
The file was built for DuckDB version 'v1.1.3', but we can only load extensions built for DuckDB version '19864453f7'.
---
Backtrace:
    ▆
 1. ├─DBI::dbExecute(con, "INSTALL postgres;")
 2. └─DBI::dbExecute(con, "INSTALL postgres;")
 3.   ├─DBI::dbSendStatement(conn, statement, ...)
 4.   └─DBI::dbSendStatement(conn, statement, ...)
 5.     ├─DBI::dbSendQuery(conn, statement, ...)
 6.     └─duckdb::dbSendQuery(conn, statement, ...)
 7.       └─duckdb (local) .local(conn, statement, ...)
 8.         └─duckdb:::duckdb_result(connection = conn, stmt_lst = stmt_lst, arrow = arrow)
Run rlang::last_trace(drop = FALSE) to see 14 hidden frames.
```
```R
rlang::last_trace(drop = FALSE)
```
```console
<error/rlang_error>
Error in `duckdb_result()`:
! rapi_execute: Failed to run query
Error: IO Error: Failed to install 'postgres_scanner'
The file was built for DuckDB version 'v1.1.3', but we can only load extensions built for DuckDB version '19864453f7'.
---
Backtrace:
     ▆
  1. ├─DBI::dbExecute(con, "INSTALL postgres;")
  2. ├─DBI::dbExecute(con, "INSTALL postgres;")
  3. │ ├─DBI::dbSendStatement(conn, statement, ...)
  4. │ └─DBI::dbSendStatement(conn, statement, ...)
  5. │   ├─DBI::dbSendQuery(conn, statement, ...)
  6. │   └─duckdb::dbSendQuery(conn, statement, ...)
  7. │     └─duckdb (local) .local(conn, statement, ...)
  8. │       └─duckdb:::duckdb_result(connection = conn, stmt_lst = stmt_lst, arrow = arrow)
  9. │         └─duckdb:::duckdb_execute(res)
 10. │           └─duckdb:::rethrow_rapi_execute(...)
 11. │             ├─rlang::try_fetch(...)
 12. │             │ ├─base::tryCatch(...)
 13. │             │ │ └─base (local) tryCatchList(expr, classes, parentenv, handlers)
 14. │             │ │   └─base (local) tryCatchOne(expr, names, parentenv, handlers[[1L]])
 15. │             │ │     └─base (local) doTryCatch(return(expr), name, parentenv, handler)
 16. │             │ └─base::withCallingHandlers(...)
 17. │             └─duckdb:::rapi_execute(stmt, arrow, integer64)
 18. └─base::.handleSimpleError(...)
 19.   └─rlang (local) h(simpleError(msg, call))
 20.     └─handlers[[1L]](cnd)
 21.       └─duckdb:::rethrow_error_from_rapi(e, call)
 22.         └─rlang::abort(msg, call = call)
>
```

### OS:

LinuxMint x86_64

### DuckDB Version:

1.1.3

### DuckDB Client:

R 4.4.2

### Full Name:

zcatav

### Affiliation:

Heartsapiens

### What is the latest build you tested with? If possible, we recommend testing with the latest nightly build.

I have not tested with any build

### Did you include all relevant data sets for reproducing the issue?

No - Other reason (please specify in the issue body)

### Did you include all code required to reproduce the issue?

- [X] Yes, I have

### Did you include all relevant configuration (e.g., CPU architecture, Python version, Linux distribution) to reproduce the issue?

- [X] Yes, I have

**Comment by kent37** [comment]
Appears to be a duplicate of #623

**Comment by krlmlr** [comment]
Closing as duplicate of #623. Will fix in duckdb 1.1.3-1.
